hdf5.spec

```
@@ -614,6 +608,9 @@ done
 %endif
 
 %changelog
+* Mon Jan 25 2021 Maureen Jean <maureen.jean@intel.com> - 1.13.0-rc5
+- Update to tagged release hdf5-1_13_0-rc5
```

Should the proper version number here, not what they name the file:
```suggestion
- Update to tagged release hdf5-1.13.0~rc5
```